src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )
+    }
+    <p className="Person__partner">
+      {person.isMarried === false
+        ? 'I am not married'
+        : `${person.partnerName} is my ${person.sex === 'm'
+          ? 'wife' : 'husband'}`}
```

Comment:
Try to avoid nested ternary, for example you can create separate variable for this expression 
`person.sex === 'm' ? 'wife' : 'husband'`